src/invidious/views/preferences.ecr

```diff
@@ -106,6 +106,12 @@
                 <input name="annotations" id="annotations" type="checkbox" <% if preferences.annotations %>checked<% end %>>
             </div>
 
+            <div class="pure-control-group">
+                <label for="extend_desc"><%= translate(locale, "Automatically extend video descriptions: ") %></label>
```

Comment:
You forgot to add the key "Automatically extend video descriptions: "  to the locales files.
Also, imho it would be better to have "video description" instead of "description**s**", as the user will never expand more than one description per video.

Reply:
Oh oops that was from the original PR. Should be fixed now